ultranest/plot.py

```diff
@@ -118,22 +118,28 @@ def get_line(self, q=0.5):
         assert len(self.ys) > 0, self.ys
         return scipy.stats.mstats.mquantiles(self.ys, q, axis=0)[0]
 
-    def shade(self, q=0.341, **kwargs):
+   def shade(self, q=0.341, ax=None, **kwargs):
         """Plot a shaded region between 0.5-q and 0.5+q. Default is 1 sigma."""
         if not 0 <= q <= 0.5:
-            raise ValueError("quantile distance from the median, q, must be between 0 and 0.5, not %s. For a 99%% quantile range, use q=0.48." % q)
+            raise ValueError("quantile distance from the median, q, must be between 0 and 0.5, not %s. For a 99% quantile range, use q=0.48." % q)
         shadeargs = dict(self.shadeargs)
         shadeargs.update(kwargs)
         lo = self.get_line(0.5 - q)
         hi = self.get_line(0.5 + q)
-        return plt.fill_between(self.x, lo, hi, **shadeargs)
+        if ax is None:
```

Comment:
Could you change here to 
```
if ax is None:
     ax = plt
```
so that the line is simpler. Also below in the next function.

Reply:
Oh yeah nice idea to make it a one liner